006_plug_and_play_prior/toto.ipynb

```diff
@@ -0,0 +1,705 @@
+      "text/plain": [
+       "<Figure size 1000x1000 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Observation\n",
```

Comment:
I agree this part is quite confusing, particularly because we define some prior and likelihood here but define a different prior and likelihood again in forming the Bayesian problem.

I changed the code to 
```python
y_exact = A_op(im)
Y_generation = Gaussian(im.flatten(), sigma2, geometry = geometry)
y_data = Y_generation.sample()
```
and added a comment saying "A noisy observation is then generated by leveraging the `sample` function of `Gaussian`."